packages/spor-bottom-drawer-react-native/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@vygruppen/spor-bottom-drawer-react-native",
```

Comment:
Jeg ville vurdert å kalle denne `spor-drawer-react-native` eller tilogmed `spor-modal-react-native` for å holde oss til samme navneparadigme som React-implementasjonen.